Gruntfile.js

```diff
@@ -181,5 +169,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-browserify');
 
   grunt.registerTask('default', ['exec:bower_update','clean','copy:fonts','sass:dev','browserify:dev','exec:build_sphinx','connect','open','watch']);
-  grunt.registerTask('build', ['exec:bower_update','clean','copy:fonts','sass:build','browserify:build','uglify','exec:build_sphinx']);
+  grunt.registerTask('build',   ['clean','copy:fonts','sass:build','browserify:build','uglify']);
+  grunt.registerTask('install', ['exec:bower_update','clean','copy:fonts','sass:build','browserify:build','uglify']);
```

Comment:
What's the reason for the new `install` step? I think we should have a single command that produces a built theme with CSS and JS, completely from scratch. It looks like you're doing that with `install`, but what would we need `build` for then?